src/data/sever-request.js

```diff
@@ -0,0 +1,10 @@
+import axios from "axios";
+
+export const getProductFromMockServer = async(setData) => {
+    try {
+        const response = await axios.get("/api/products");
+        setData(response?.data?.products);
+    }catch(e) {
+        console.log(e);
```

Comment:
suggestion: use console.error(e) for displaying error messages

Reply:
Done